manifests/concourse-manifest/extensions/datadog.yml

```diff
@@ -1,3 +1,9 @@
+releases:
+  - name: riemann
+    url: https://s3-eu-west-1.amazonaws.com/michael-state/riemann-2%2Bdev.1.tgz
```

Comment:
Could we stash this somewhere that isn't associated with a particular environment?